services/videoDownloader.py

```diff
@@ -15,14 +15,17 @@ def getVideoDuration(videoPath):
   video = VideoFileClip(videoPath)
   return video.duration
 
+def getVideoTitle(vidUrl):
```

Comment:
Should we make a video title field in the table?

Reply:
We stopped doing this somewhere along the way. I think it is extremely helpful to have a video title to be able to query by name.